setup

```diff
@@ -177,16 +177,15 @@ def run_build_action(copy_dist_dest_path: str):
 
 
 def run_modules_storage_setup_action(modules_storage_path_param):
-    if modules_storage_path_param is None:
-        modules_storage_path = f"{pathlib.Path().resolve()}/dist"
-    else:
-        modules_storage_path = modules_storage_path_param
+    modules_storage_path = f"{pathlib.Path().resolve()}/dist" if modules_storage_path_param is None else modules_storage_path_param
 
     log.log(level=log.INFO,
             msg="--query-modules-dir flag in {memgraph_conf} will be set to {path}".format(path=modules_storage_path,
-                                                                                       memgraph_conf=MG_CONF_PATH))
+                                                                                           memgraph_conf=MG_CONF_PATH))
 
-    subprocess.run(['sed', '-E', '-i', "s@.*(--query-modules-directory).*@\\1={path}@".format(path=modules_storage_path), MG_CONF_PATH  ])
+    subprocess.run(
+        ['sed', '-E', '-i', "s@.*(--query-modules-directory).*@\\1={path}@".format(path=modules_storage_path),
+         MG_CONF_PATH])
```

Comment:
We should talk about using `sed` here. Or discuss using bash script for the whole process

Reply:
Yes of course, what do you mean?

Reply:
Yes of course, what do you mean?